lib/health_monitor/rake_task.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require "rake"
+
+class RakeTask
+  include Rake::DSL if defined? Rake::DSL
+
+  def install_tasks
+    desc "Check if Sidekiq is running"
+    task "health_monitor:sidekiq:health" => :environment do
+      hostname = ENV["HOSTNAME"] || `hostname -f`.strip
+      ps = Sidekiq::ProcessSet.new
+      up = ps.any? { |p| p["hostname"] == hostname && p["quiet"] == "false" }
+
+      if up
+        puts "UP"
+        exit 0
+      end
+
+      puts "DOWN"
```

Comment:
the puts are necessary for OS ?

Reply:
This is better for Debugging, if we want to investigate if there is something happening with the script inside an OS POD. The OS Shell acts weird sometimes. It is also good if there is any logging on the Health Check, we could see what and when it happened in our Logging view interface.